docs/guides/guides/screenshots-and-videos.mdx

```diff
@@ -79,46 +75,54 @@ clear your videos folder before a run, you can set
 
 ### Video encoding
 
-If your spec files have a long run duration, you might notice a time gap between
-a finished spec and a new spec starting during `cypress run`. During this time,
-Cypress is encoding the captured video and possibly uploading it to Cypress
-Cloud.
+After a video is recorded, Cypress encodes the video to a commonly digestable
+format. Part of this encoding process includes video compression.
 
-If the machine is encoding the video slowly (which is often the case for virtual
-machines that use less CPU cores), the encoding might take a long time. In this
-case, you can modify the
-[`videoCompression`](/guides/references/configuration#Videos) configuration to
-make the encoding a little bit faster. Here are some common scenarios:
+Compression is disabled by default, meaning this step will be skipped
+completely, so the file size of the video will be larger, but the encoding
+process should be faster. Setting
+[`videoCompression`](/guides/references/configuration#Videos) to `true` will
+coerce the value to 32 CRF, which takes longer to process, but results in a
+smaller video.
 
-**Change compression value**
+**Enabling compression**
 
 :::cypress-config-example
 
 ```ts
 {
-  videoCompression: 15
+  videoCompression: true
 }
 ```
 
-:::cypress-config-example
+:::
 
-A lower `videoCompression` value will spend less time compressing and result in
-a bigger video file size.
+If your spec files have a long run duration and
+[`videoCompression`](/guides/references/configuration#Videos) is enabled, you
+might notice a time gap between a finished spec and a new spec starting during
+`cypress run`. During this time, Cypress is encoding the captured video and
+possibly uploading it to Cypress Cloud.
 
-**Disable compression**
+If the machine is encoding the video slowly (which is often the case for virtual
+machines that use less CPU cores), the encoding might take a long time. In this
```

Comment:
messy suggestion but your current paragraph hangs. /  doesn't list common scenarios:
```suggestion
**Change compression value from 32**

In addition to enabling or disabling video compress, you can specify the CRF value used to compress the video. Here is some common scenario:

- If the machine is encoding the video slowly (which is often the case for virtual
machines that use less CPU cores), increase(??) the CRF value to .....

- If your videos are extremely low quality, try decreasing the CRF value....

```

Reply:
gave this a shot in 206a57c0a